AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -4412,6 +4412,7 @@ wyborcza.pl,wyborcza.biz##.mod_ddbox[data-cta-category]
 youtube.com##.ytd-single-option-survey-option-renderer
 indianexpress.com##.bell-icon
 blog.livedoor.com##a[href="http://blog.livedoor.com/influencernews/"]
+blog.livedoor.com##a[href="https://www.youtube.com/@YoMuRy-mangadouga"]
```

Comment:
I would narrow this rule, and please add to the top of the list, above this rule: `fox8live.com##.powa-sticky > div.powa-sticky`
blog.livedoor.com##body > a[href="https://www.youtube.com/@YoMuRy-mangadouga"]